docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -97,93 +196,231 @@ Windows::
+The output is similar to the following
++
 [source,shell]
 ----
-$ clc -c target.yaml map size --name my-map
 1000
 OK
 ----
 
+You can also check a random value from the data we populated in the xref:migrate:data-migration-tool.adoc#add-data-to-cluster[Add Data to Cluster] section above using the following command:
+
+[source,shell]
+----
+clc -c target.yaml map get key-42 --name my-map
+----
+
+The output is similar to the following:
+
 [source,shell]
 ----
-$ clc -c target.yaml map get key-42 --name my-map
 value-42
 OK
 ----
 
-The `target.yaml` file is used by Hazelcast CLC to connect to the target cluster and verify the data is migrated.
-
-Alternatively, you can use the https://docs.hazelcast.com/management-center/latest/getting-started/install[Hazelcast Management Center] to verify the data in the target cluster.
+[view-result]
+=== View Migration Details
 
-== Using the DMT for an Existing Cluster
+When the migration completes, details of the migration are created in the following:
 
-To migrate data from your existing cluster (source), skip Step 1 and Step 2 in <<trying-out-the-dmt, Trying Out the DMT>> above.
+* Migration report
++
+This is written to the _migration$$_$$report$$_$$[migration_id].txt_ file in the directory used when running the `dmt` command.
 
-Before starting with Step 3 and moving forward, you need to
+* DMT log file
++
+This is the file specified in the `--log.path` flag of the `start` command.
++
+If the flag is not used, the file is saved to the location set in the `CLC_HOME` environment variable. If this environment variable is not set, the default location is the _~/.hazelcast_ folder.
++
+NOTE: Logging uses the same environment variables as Hazelcast CLC. For further information on environment variables, refer to the https://docs.hazelcast.com/clc/latest/environment-variables[Environment Variables^] section of the Hazelcast CLC documentation.
++
+The DMT log file includes migration member logs and other DMT logs.
++
+The logs are in the format `[(migration_id)_(member uuid)] (member log)`
 
-* Check the `migration_config/source/hazelcast-client.yaml` file in the folder where you extracted the DMT package.
-Modify the `cluster-name` and `cluster-members` fields in this file, such that they match the name of your existing cluster and addresses of the cluster members.
-* Check the `migration_config/data/imap_names.txt` and `migration_config/data/replicated_map_names.txt` files in the folder where you extracted the DMT package.
-Modify the content of these files, such that they match the names of your existing maps and replicated maps. If you have multiple maps/replicated maps, put one map name per line.
\ No newline at end of file
+* `__datamigration_results` IMap
++
+This is created on the target cluster.
++
+The keys are UUID4 string format migration IDs, and the values are `HazelcastJsonValue` serialization interfaces that correspond to migration statuses.
++
+The migration report is also included as a field.
```

Comment:
After a second thought, I think this sentence can stay. Also let's add the following to the sentence above: A migration status represent a completed migration's details and can be provided to Hazelcast Support for getting help.